app/src/main/java/org/openobservatory/ooniprobe/common/AppDatabase.java

```diff
@@ -11,7 +11,7 @@
 @Database(name = AppDatabase.NAME, version = AppDatabase.VERSION, foreignKeyConstraintsEnforced = true)
 public class AppDatabase {
     public static final String NAME = "v2";
-    public static final int VERSION = 3;
+    public static final int VERSION = 3 + 2;
```

Comment:
?

Reply:
I am adding `+2` here so i don't forget why the increment is 2. first increment is for modifications done to the schema ,`Migration4` and another for the addition of a new model `TestDescriptor`